h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1205,7 +1216,8 @@ public GLMOutput(GLM glm) {
         _random_column_names = Arrays.copyOf(glm._randomColNames, glm._randomColNames.length);
       }
       _coefficient_names[_coefficient_names.length-1] = "Intercept";
-      _binomial = (glm._parms._family == Family.binomial || glm._parms._family == Family.quasibinomial);
+      _binomial = (glm._parms._family == Family.binomial || glm._parms._family == Family.quasibinomial || 
+              glm._parms._family.equals(Family.fractionalbinomial));
```

Comment:
```suggestion
      _binomial = (glm._parms._family == Family.binomial || glm._parms._family == Family.quasibinomial ||
              Family.fractionalbinomial == glm._parms._family);
```
Compile-time checks on sanity of the enum comparison + removed a trailing space.